### ISSUE TITLE
Packaging library with component with external less stylesheet from working directory with spaces

### ISSUE THREAD

**Issue by daryl0313** [ng-packagr/ng-packagr#1197]
## Type of Issue

```
[X] Bug Report
[ ] Feature Request
```

## Description

I am getting the issue below when trying to compile am angular library which has a component and uses an external `less` file for styles.  I think this may be windows specific and appears to be caused by the fact that my working directory has a space in the path, i.e. `C:\My Path\With Spaces\my-project`.

```
Building Angular Package
Building entry point 'mylibrary'
Compiling TypeScript sources through ngc
lessc: ENOENT: no such file or directory, open 'C:\My'

BUILD ERROR
: Error: Command failed: node node_modules/less/bin/lessc C:\My Path\With Spaces\my-project\libraries\mylibrary\src\lib\my-component\my-component.component.less --less-plugin-npm-import="prefix=~"
lessc: ENOENT: no such file or directory, open 'C:\My'

    at checkExecSyncError (child_process.js:601:13)
    at Object.execSync (child_process.js:641:13)
    at StylesheetProcessor.renderPreProcessor (C:\My Path\With Spaces\my-project\node_modules\ng-packagr\lib\ng-v5\entry-point\resources\stylesheet-processor.js:56:40)
    at StylesheetProcessor.process (C:\My Path\With Spaces\my-project\node_modules\ng-packagr\lib\ng-v5\entry-point\resources\stylesheet-processor.js:25:34)
    at Object.readResource (C:\My Path\With Spaces\my-project\node_modules\ng-packagr\lib\ts\cache-compiler-host.js:68:57)
    at TsCompilerAotCompilerTypeCheckHostAdapter.loadResource (C:\My Path\With Spaces\my-project\node_modules\@angular\compiler-cli\src\transformers\compiler_host.js:477:37)
    at Object.get (C:\My Path\With Spaces\my-project\node_modules\@angular\compiler\bundles\compiler.umd.js:23177:94)
    at DirectiveNormalizer._fetch (C:\My Path\With Spaces\my-project\node_modules\@angular\compiler\bundles\compiler.umd.js:2465:47)
    at C:\My Path\With Spaces\my-project\node_modules\@angular\compiler\bundles\compiler.umd.js:2586:72
    at Array.map (<anonymous>)
```

#### How To Reproduce

1. Create a working directory with spaces in it, i.e. `C:\My Path\With Spaces\my-project`.
2. Create an angular library project.
3. Create a component in library project which references an external `less` style file.
4. Run `ng build` to build library.

#### Expected Behaviour

Should build library with no exceptions.

#### Version Information

```
$ node_modules/.bin/ng-packagr --version
ng-packagr: 4.4.5
@angular/*: 6.1.10
typescript: 2.8.3
rxjs: 6.3.3
node: 8.11.1
npm: 5.8.0
```

**Comment by daryl0313** [comment]
The issue seems to be related to [this line](https://github.com/ng-packagr/ng-packagr/blob/2eac5bf83e91f4ddf752560774b03381fd7a51a9/src/lib/ng-v5/entry-point/resources/stylesheet-processor.ts#L59).  Wrapping the fullPath argument in quotes fixes this issue.

**Comment by alan-agius4** [comment]
Can you land a pr please?

Thanks

**Comment by alan-agius4** [comment]
The fix is available in version 4.7.0
